test/integration/services/data/test-insert-offender-manager.js

```diff
@@ -32,7 +32,7 @@ describe('app/services/data/insert-offender-manager', function () {
           expect(result['forename']).to.be.null // eslint-disable-line
           expect(result['surname']).to.be.null // eslint-disable-line
           expect(result['type_id']).to.eq(typeId) // eslint-disable-line
-          expect(moment().diff(result['effective_from'], 'seconds')).to.be.lt(10) // eslint-disable-line
+          expect(moment().diff(result['effective_from'], 'seconds')).to.be.lt(20) // eslint-disable-line
```

Comment:
Did this sort the failing tests?

Reply:
Means they fail less often, haven't had any fails in a while now can change back if needed